src/ontology/mondo-edit.obo

```diff
@@ -347771,7 +347773,7 @@ is_a: MONDO:0016153 {source="OMIM:615988"} ! neuromuscular disease caused by qua
 intersection_of: MONDO:0015229 ! Bardet-Biedl syndrome
 intersection_of: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/16380 ! TRIM32
 relationship: has_characteristic HP:0000007 {source="MONDO:HPOA", source="OMIM:615988"} ! Autosomal recessive inheritance
-relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/16380 {source="MONDO:mim2gene_medgen", source="OMIM:615988"} ! TRIM32
+relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/16380 {source="MONDO:mim2gene_medgen"} ! TRIM32
```

Comment:
The gene association is marked as provisional, phenotype name starts with `?`, https://omim.org/entry/615988, and therefore does not meet the criteria to be maintained by the omim ingest pipeline. Please confirm if this needs to be maintained and added to the "protected" file.

Reply:
This source should be maintained and added to the "protected" file